bee-node/bee-plugin/src/lib.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+//! The bee node API for plugins.
+
+#![deny(missing_docs, warnings)]
+
+mod handler;
+mod manager;
+mod streamer;
+mod grpc {
+    tonic::include_proto!("plugin");
+}
+mod error;
+mod plugin;
+
+pub mod event;
+pub mod handshake;
+pub mod hotloading;
+
+pub use error::PluginError;
+pub use manager::PluginManager;
+pub use plugin::{serve_plugin, Plugin};
+
+use std::any::TypeId;
+
+/// A unique identifier for `EventBus` callbacks.
+#[derive(Clone, Copy, PartialEq, Eq)]
+pub enum UniqueId {
```

Comment:
I still think that this type could be useful in the bee-event-bus crate. With variants `Type` and `Object` maybe.